homeassistant/components/modbus/sensor.py

```diff
@@ -105,91 +43,12 @@ async def async_setup_platform(
     """Set up the Modbus sensors."""
     sensors = []
 
-    #  check for old config:
-    if discovery_info is None:
```

Comment:
Please leave the "if" with just a return, to safeguard.

Reply:
👍🏼